File: Data Structures & Algorithms/concatenation-of-array/submission-0.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int total = nums.Length;
        int[] result = new int[total * 2];

        for(int i = 0; i<nums.Length; i++){
            result[i] = nums[i];
            result[i+total] = nums[i];
        }
        return result;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        p1 = 0 
        p2 = len(nums)

        while p1 < p2:
            if nums[p1] == val:
                p2 -= 1
                nums[p1] = nums[p2]
            else:
                p1 += 1
                
        return p2